README.md

```diff
@@ -56,20 +53,23 @@ let res = dynamo_client
 Creating new span:
 
 ```rust
-// create new span directly by using current span context
-let aws_span = AwsSpan::new(AwsTarget::Dynamo("table_name"), "GetItem");
+// create new span in the current span's context using either a dedicated constructor
+let aws_span = DynamoDBOperation::get_item("table_name").start();
+// or a generic one
+let aws_span = DynamoDBOperation::new("GetItem", vec!["table_name"]).start();
```

Comment:
I left a way to pass a custom string as well, though I'm not sure if it is actually needed here.